src/DIPS.Xamarin.UI/Controls/FloatingActionMenu/FloatingActionMenuBehaviour.cs

```diff
@@ -96,8 +97,25 @@ private static void IsVisiblePropertyChanged(BindableObject bindable, object old
         {
             if (bindable is FloatingActionMenuBehaviour behaviour)
             {
-                behaviour.Children.ForEach(mb => mb.IsVisible = (bool) newvalue);
-                behaviour.Children.ForEach(mb => mb.IsBadgeVisible = (bool)newvalue);
+                behaviour.Children.ForEach(mb => mb.IsVisible = (bool)newvalue);
+
+                var menuButton = behaviour.Children.FindAll(mb => mb.Title == "Notifications");
```

Comment:
This won't do. Can not do a check on a specific title. What is the bug here?

Reply:
In document view if rotate from Landscape mode to Portrait can see empty badge

Reply:
A badge can be set on any menubutton, so you will have to iterate over all of them and toggle badge visibility based on that. Remember that this is a public package, we should have no DIPS business logic in the code

Reply:
Ok  done some change, can you check it